src/scripts/dungeons/Dungeon.ts

```diff
@@ -363,6 +363,17 @@ dungeonList['Sky Pillar'] = new Dungeon('Sky Pillar',
     ],
     40000, 101, 5);
 
+dungeonList['Sealed Cave'] = new Dungeon('Sealed Cave',
+    ['Zubat', 'Poochyena', 'Nosepass'],
```

Comment:
These don't seem to be the correct Pokemon for the cave:
![image](https://user-images.githubusercontent.com/7288322/94082888-01428280-fe56-11ea-866b-cc42841b94d2.png)

Reply:
Fixed